README.md

```diff
@@ -210,6 +210,22 @@ N/A
     signal: start
 ```
 
+### Using a Private Instance of Healthchecks.io
+
+The `api_base_url` variable allows you to specify the custom API URL of your healthchecks.io instance, which can be used to connect to your private instance.
```

Comment:
Nit: perhaps capitalize `healthchecks.io` to match the heading?

Reply:
The language is a bit awkward, perhaps we write this as:

### Using a self-hosted instance of Healthchecks.io

The `api_base_url` parameter can be used to direct the modules in this Collection towards a self-hosted instance of Healthchecks.io. By default, or if unset, it defaults to the public instance located at [https://hc-ping.com](https://hc-ping.com).